app/controllers/search_controller.rb

```diff
@@ -1,4 +1,6 @@
 class SearchController < ApplicationController
+  before_action :require_user
```

Comment:
if we're calling this in every controller now, should we just move it to ApplicationController??

Reply:
Every controller minus Sessions and Welcome because those need to be available to visitors